doc/nrf/ug_partition_manager.rst

```diff
@@ -82,6 +82,113 @@ span: list
    When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
    This property cannot be used together with the `placement` property.
 
+region: string
+   The memory region (address space) this partition resides in.
+   The placement of partitions is solved separately for each region.
+   The runtime and lifetime of a region is not solved separately.
+   Also, the regions share the same namespace for partition names.
+
+runtime: dict
+   When this partition is active (running).
+   Only image partitions can have a runtime.
+   The valid keywords are:
+
+      before: list
+         This partition is active before all partitions in the list.
+         This typical indicates that it boots those partitions,
+         but it doesn't indicate that the partitions in the list run concurrently (this must be indicated explicitly).
+
+      after: list
+         This partition is active after all partitions in the list.
+         This typical indicates that it is booted by one of those partitions.
+
+      alongside: list
+         This partition is active at the same time as all partitions in the list.
+         This indicates that they run concurrently, and that they must share the
+         available space in all regions.
+         This list will also inherit all the `alongside` entries of the partitions
```

Comment:
`This list will also inherit` - does not make sense IMO. Either the list is modified, or the items in the list is modified. In this case I think it would be better to say `This list is extended with ...`

Reply:
Thanks, will update the text.

Reply:
Done